ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -301,28 +278,50 @@ objects:
       - -c
       - |
         #!/bin/bash
+        set -euo pipefail
         function teardown() {
+          # This is for running the gcloud commands
+          mock-nss.sh
+          gcloud auth activate-service-account \
+            --quiet --key-file /tmp/cluster/gce.json
+          gcloud --quiet config set project "${GOOGLE_PROJECT_ID}"
+          gcloud --quiet config set compute/zone "${GOOGLE_COMPUTE_ZONE}"
+
+          set -x
           set +e
+          echo 'Collect all the info about clusteroperators'
+          LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
+            --zone "${GOOGLE_COMPUTE_ZONE}" \
+            packer@"${INSTANCE_PREFIX}" \
+            --command 'export KUBECONFIG=/home/$USER/clusters/nested/auth/kubeconfig && bash -ce "oc get co"' | tee /tmp/artifacts/output-co-libvirt
+          echo 'Run must gather on the cluster'
+          LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
+            --zone "${GOOGLE_COMPUTE_ZONE}" \
+            packer@"${INSTANCE_PREFIX}" \
+            --command 'mkdir -p $HOME/clusters/nested/must-gather-logs && export KUBECONFIG=$HOME/clusters/nested/auth/kubeconfig && bash -ce "oc adm must-gather --dest-dir $HOME/clusters/nested/must-gather-logs || true"'
```

Comment:
Nit: `must-gather-logs` -> `must-gather`, to match out other templates and create a more familiar environment for folks switching between platforms.